nafas/__main__.py

```diff
@@ -20,7 +21,8 @@ def main() -> None:
         parser.add_argument('--version', help='version', nargs="?", const=1)
         parser.add_argument('--silent', help='silent mode', nargs="?", const=1)
         parser.add_argument('--skip-intro', help='skip intro', nargs="?", const=1)
-        parser.add_argument('--config', help='path to the configuration file', type=str)
+        parser.add_argument('--init-config', help='generate a starter configuration file at the given path', type=str)
```

Comment:
They name can be confusing ig (this may be taken as "initiate config from"). I'd suggest having `--gen-config`.

Reply:
Yeah that's better.